source/Octostache.Tests/UsageFixture.cs

```diff
@@ -91,10 +91,9 @@ public void EmptyValuesAreEchoed(string template, string variableDefinitions, st
         [Test]
         [TestCase("#{Foo}", "Foo=#{Foo}")]
         [TestCase("#{Foo}", "Foo=#{Fox};Fox=#{Fax};Fax=#{Fix};Fix=#{Foo}")]
-        [ExpectedException(typeof(InvalidOperationException))]
         public void MaximumRecursionLimitException(string template, string variableDefinitions)
         {
-            ParseVariables(variableDefinitions).Evaluate(template);
+            Assert.That(() => ParseVariables(variableDefinitions).Evaluate(template), Throws.InvalidOperationException);
```

Comment:
Not convinced - bringing in a full library for just a single assertion doesn't sound great.